src/java/org/apache/cassandra/io/sstable/IndexInfo.java

```diff
@@ -59,7 +59,7 @@
  */
 public class IndexInfo
```

Comment:
Not related to the patch, but `import org.apache.cassandra.db.marshal.ByteArrayAccessor;` is unused ^